org.eclipse.wb.core.java/src/org/eclipse/wb/internal/core/nls/bundle/pure/activator/ActivatorSourceNewComposite.java

```diff
@@ -0,0 +1,99 @@
+/*******************************************************************************
+ * Copyright (c) 2019-2020
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *    Google, Inc. - initial API and implementation
+ *    bergert - modified to implement Activator.getPluginBundle().getString()
+ *******************************************************************************/
+package org.eclipse.wb.internal.core.nls.bundle.pure.activator;
+
+import org.eclipse.wb.core.model.JavaInfo;
+import org.eclipse.wb.internal.core.nls.Messages;
+import org.eclipse.wb.internal.core.nls.bundle.AbstractBundleSource;
+import org.eclipse.wb.internal.core.nls.bundle.AbstractBundleSourceNewComposite;
+import org.eclipse.wb.internal.core.nls.edit.IEditableSource;
+import org.eclipse.wb.internal.core.utils.reflect.ReflectionUtils;
+
+import org.eclipse.swt.widgets.Composite;
+
+/**
+ * Composite for creating new source.
+ *
+ * @author bergert
+ * @coverage core.nls.ui
+ */
+public final class ActivatorSourceNewComposite extends AbstractBundleSourceNewComposite {
+  ////////////////////////////////////////////////////////////////////////////

```

Comment:
Old style comments. There is no need to copy that coding style.

Reply:
I copied from the original code .... ok